### PR TITLE
remove leading and ending quotes for service test csv column value which contains comma

### PR DESCRIPTION
**Pull request by YannanGao-gs**
#### What type of PR is this?

Improvement

#### What does this PR do / why is it needed ?

remove leading and ending quotes for service test csv column value which contains comma

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by YannanGao-gs** [comment]
@MauricioUyaguari  workaround for one simple and naive case needed by users

File: legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/helperFunctions/helperFunctions.pure [hunk 1]
```diff
@@ -116,6 +116,7 @@ function <<access.private>> meta::alloy::service::execution::splitWithEmptyValue
 
    $s->meta::alloy::service::execution::replaceWithEmptyValue($delimForEmptyCsvField)
         ->meta::alloy::service::execution::splitCsvRow(',')
+        ->map(csv_val| if($csv_val->contains(',') && $csv_val->startsWith('"') && $csv_val->endsWith('"'), | $csv_val->substring(1, $csv_val->length()-1) ,| $csv_val))
```
Comment (MauricioUyaguari) [inline legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/helperFunctions/helperFunctions.pure:119]: can we add a test for the csv to sql statements to ensure the expected values are carried through?

Comment (YannanGao-gs) [reply legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/helperFunctions/helperFunctions.pure:119]: > also confused why we didnt have to update tests from this PR #1988

The test added in PR https://github.com/finos/legend-engine/pull/1988 has a leading white space. We don't remove white space.

Comment (MauricioUyaguari) [reply legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/helperFunctions/helperFunctions.pure:119]: yup just saw that, lets add a test where we ensure this. 
what about `testRelationalServiceWithCommmaInCSV()` in https://github.com/finos/legend-engine/pull/1988/
would expect that to start failing or change?

File: legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/testDDL.pure [hunk 2]
```diff
@@ -153,4 +153,45 @@ function <<test.Test>>  meta::relational::tests::ddl::testSetupDataSqlGeneration
    'insert into personTable (ID,FIRSTNAME,LASTNAME,AGE,ADDRESSID,FIRMID,MANAGERID) values (1,\' Peter\',\'  "I\'\'m Smith, Jr"\',23,1,1,2);'
    ];
   assertSameElements($sqls,$expectedSqls);
+}
```
Comment (MauricioUyaguari) [inline legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/testDDL.pure:156]: can't we just add another row of values to the previous test ?

**Pull request merged**